presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -88,7 +88,8 @@ public void testSTArea()
     {
         assertFunction("ST_Area(ST_GeometryFromText('POLYGON ((2 2, 2 6, 6 6, 6 2))'))", DOUBLE, 16.0);
         assertFunction("ST_Area(ST_GeometryFromText('POLYGON EMPTY'))", DOUBLE, 0.0);
-        assertInvalidFunction("ST_Area(ST_GeometryFromText('POINT (1 4)'))", "ST_Area only applies to POLYGON or MULTI_POLYGON. Input type is: POINT");
+        assertFunction("ST_Area(ST_GeometryFromText('POINT (1 4)'))", DOUBLE, 0.0);
```

Comment:
Could you tests for an empty point, an empty line and a few geometry collections?

nit: consider extracting a helper method `assertArea`

```
private void assertArea(String wkt, double expectedArea)
{
   assertFunction(format("ST_Area(ST_GeometryFromText('%s'))", wkt), DOUBLE, expectedArea);
}
```